tests/Integration/Controller/ListControllerTest.php

```diff
@@ -320,4 +322,63 @@ public function getListMembersWithCurrentSessionKeyForExistingListWithSubscriber
             ]
         );
     }
+
+    /**
+     * @test
+     */
+    public function getListNumberForExistingListWithoutSessionKeyReturnsForbiddenStatus()
+    {
+        $this->getDataSet()->addTable(static::LISTS_TABLE_NAME, __DIR__ . '/Fixtures/SubscriberList.csv');
+        $this->applyDatabaseChanges();
+
+        $this->client->request('get', '/api/v2/lists/1/number');
+
+        $this->assertHttpForbidden();
+    }
+
+    /**
+     * @test
+     */
+    public function getListNumberForExistingListWithExpiredSessionKeyReturnsForbiddenStatus()
+    {
+        $this->getDataSet()->addTable(static::LISTS_TABLE_NAME, __DIR__ . '/Fixtures/SubscriberList.csv');
+        $this->getDataSet()->addTable(static::ADMINISTRATOR_TABLE_NAME, __DIR__ . '/Fixtures/Administrator.csv');
+        $this->getDataSet()->addTable(static::TOKEN_TABLE_NAME, __DIR__ . '/Fixtures/AdministratorToken.csv');
+        $this->applyDatabaseChanges();
+
+        $this->client->request(
+            'get',
+            '/api/v2/lists/1/number',
+            [],
+            [],
+            ['PHP_AUTH_USER' => 'unused', 'PHP_AUTH_PW' => 'cfdf64eecbbf336628b0f3071adba763']
+        );
+
+        $this->assertHttpForbidden();
+    }
+
+    /**
+     * @test
+     */
+    public function getListNumberWithCurrentSessionKeyForExistingListReturnsOkayStatus()
+    {
+        $this->getDataSet()->addTable(static::LISTS_TABLE_NAME, __DIR__ . '/Fixtures/SubscriberList.csv');
+        $this->applyDatabaseChanges();
+
+        $this->authenticatedJsonRequest('get', '/api/v2/lists/1/number');
+
+        $this->assertHttpOkay();
+    }
+
+    public function getListNumberWithCurrentSessionKeyForExistingListWithSubscribersReturnsSubscribers()
```

Comment:
The test name is misleading: It doesn't return the subscribers, but the number of subscribers.

In addition to the "there are subscribes" case, we'll also need a test for the case of a list without subscribers.